docs/cue/reference/components.cue

```diff
@@ -879,6 +943,82 @@ components: {
 			}
 		}
 
+		env_vars: {
+			_http_proxy: {
+				description: """
+					The global URL to proxy HTTPS requests through.
+
+					If another HTTPS proxy is set in the configuration file or at a component level,
+					this one will be overridden.
+
+					The lowercase variant has priority over the uppercase one.
+					"""
+				type: string: {
+					default: null
+					examples: ["http://foo.bar:3128"]
+					syntax: "literal"
+				}
+			}
+			_https_proxy: {
+				description: """
+					The global URL to proxy HTTPS requests through.
+
+					If another HTTPS proxy is set in the configuration file or at a component level,
+					this one will be overriden.
+
+					The lowercase variant has priority over the uppercase one.
+					"""
+				type: string: {
+					default: null
+					examples: ["http://foo.bar:3128"]
```

Comment:
Should this be `https`?

Reply:
I don't think so, a `https` proxy can be done over `http` 🤔